# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/jsep/JsepSessionImpl.h
# Commit: d570e7369906
# Full Hash: d570e7369906010743035437ff6ff4b4c7a34e67
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2022-04-06 04:35:13
# Regressor Bug: 1624989
# File Overlap Count: 1
# Description:
#   Bug 1624989: Move RTCPeerConnection's operations chain to c++ r=jib,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D141738
# ==============================================================================

diff -r 056457fa52fe -r d570e7369906 dom/media/webrtc/jsep/JsepSessionImpl.h
--- a/dom/media/webrtc/jsep/JsepSessionImpl.h	Tue Apr 05 17:41:37 2022 +0000
+++ b/dom/media/webrtc/jsep/JsepSessionImpl.h	Tue Apr 05 18:39:12 2022 +0000
@@ -135,6 +135,9 @@
     return !mOldIceUfrag.empty();
   }
 
+  virtual std::set<std::pair<std::string, std::string>> GetLocalIceCredentials()
+      const override;
+
   virtual const std::map<size_t, RefPtr<JsepTransceiver>>& GetTransceivers()
       const override {
     return mTransceivers;